docs/en/ingest-management/agent-standalone.asciidoc

```diff
@@ -0,0 +1,56 @@
+[discrete]
```

Comment:
Note that this content is not currently being built. I plan to move it to the agent docs.